bin/test-browser.js

```diff
@@ -49,6 +49,9 @@ if (client.runner === 'saucelabs') {
   qs.saucelabs = true;
 }
 if (process.env.GREP) {
```

Comment:
heh copy and paste fail, luckily we only use INVERT when GREP is defined but good catch, thanks :)